lib/rules/prop-types.js

```diff
@@ -333,10 +333,7 @@ module.exports = {
      * @param {string} the identifier to strip
      */
     function stripQuotes(string) {
```

Comment:
Yeah - that's what I mentioned in the other PR: `prop-types.js` and `no-unused-prop-types` **on first sight** seem to share quite a bit of logic that deals with extracting the declared prop types for a component. And currently, the logic is not in sync (e.g. props in TypedArgument for Flow is not supported at all for `no-unused-prop-types`.

I'll try to deal with extracting as much of the shared logic as possible this weekend. Created an issue: https://github.com/yannickcr/eslint-plugin-react/issues/1393